packages/solo/src/start.js

```diff
@@ -144,6 +151,18 @@ const buildSwingset = async (
     plugin: { ...plugin.endowments },
   };
 
+  const env = {
+    ...process.env,
+    OTEL_EXPORTER_PROMETHEUS_PORT:
+      process.env.SOLO_OTEL_EXPORTER_PROMETHEUS_PORT,
+    OTEL_RESOURCE_ATTRIBUTES: process.env.SOLO_OTEL_RESOURCE_ATTRIBUTES,
+  };
```

Comment:
Shouldn't we also remove the `SOLO_` envs as well?

For example:

```suggestion
  const {
    SOLO_OTEL_EXPORTER_PROMETHEUS_PORT,
    SOLO_OTEL_RESOURCE_ATTRIBUTES,
    ...restEnv
  } = process.env;
  const env = {
    ...restEnv,
    OTEL_EXPORTER_PROMETHEUS_PORT: SOLO_OTEL_EXPORTER_PROMETHEUS_PORT,
    OTEL_RESOURCE_ATTRIBUTES: SOLO_OTEL_RESOURCE_ATTRIBUTES,
  };
```